packages/host/app/components/operator-mode/overlays.gts

```diff
@@ -113,23 +105,52 @@ export default class OperatorModeOverlays extends Component<Signature> {
     </style>
   </template>
 
+  @tracked isHoverOnCard = new TrackedWeakMap<RenderedLinksToCard, boolean>();
```

Comment:
Isn't there only one card that can be hovered at a time? So maybe you just need a `currentlyHoveredCard`tracked property?